recipes/gtlab-logging/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan import ConanFile
+import os
+
+required_conan_version = ">=1.59.0"
+
+class GTLabLoggingConan(ConanFile):
+    name = "gtlab-logging"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dlr-gtlab/gt-logging"
+    topics = ("logging", "qt")
+    description = "Simple logging interface with qt support"
+
+    settings = "os", "arch", "compiler", "build_type"
+    
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_qt": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_qt": False
+    }
+
+    def requirements(self):
+        if self.options.with_qt:
+            self.requires("qt/[>=5.0.0]")
+
+
+    @property
+    def _min_cppstd(self):
+        return "14"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "14": {
+                "Visual Studio": "15",
+                "msvc": "191",
+                "gcc": "7.3.1",
+                "clang": "6",
+                "apple-clang": "14",
+            },
+        }.get(self._min_cppstd, {})
+
+    def validate(self):
+        """
+        Adapted from the gtest recipe
+
+        Make sure that C++ 14 is available
+        """
+
+        if self.settings.get_safe("compiler.cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+
+        def loose_lt_semver(v1, v2):
+            lv1 = [int(v) for v in v1.split(".")]
+            lv2 = [int(v) for v in v2.split(".")]
+            min_length = min(len(lv1), len(lv2))
+            return lv1[:min_length] < lv2[:min_length]
```

Comment:
```suggestion
            return all(int(p1) < int(p2) for p1, p2 in zip(str(v1).split("."), str(v2).split(".")))
```
Can be expressed more succinctly.